static/styles/sec.css

```diff
@@ -0,0 +1,141 @@
+@import url(audit.css);
```

Comment:
Not sure how to handle this dynamically

Reply:
You don't have to, but will try to do a CSS refactoring but the stylesheets are relatively small to affect the performance.

Reply:
You don't have to, but will try to do a CSS refactoring later if I get time but the stylesheets are relatively small to affect the performance.